<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PanelSpecialElements</class>
 <widget class="QWidget" name="PanelSpecialElements">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>643</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>221</x>
     <y>31</y>
     <width>46</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Shape:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxShape">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>31</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Box</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Sphere</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Cylinder</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Cone</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Capsule</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Custom</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>190</y>
     <width>62</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Collision:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxCollision">
   <property name="geometry">
    <rect>
     <x>289</x>
     <y>190</y>
     <width>98</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>None</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Perfect</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Simplified</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Custom</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>230</y>
     <width>61</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Scale:</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>230</y>
     <width>101</width>
     <height>26</height>
    </rect>
   </property>
   <property name="decimals">
    <number>4</number>
   </property>
   <property name="value">
    <double>1.000000000000000</double>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_22">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>1</y>
     <width>199</width>
     <height>591</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="title">
    <string>Complete list</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_29">
    <item row="0" column="0">
     <widget class="PanelSuperList" name="panelSuperList" native="true">
      <property name="minimumSize">
       <size>
        <width>175</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>175</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBoxSize">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>420</y>
     <width>311</width>
     <height>37</height>
    </rect>
   </property>
   <property name="title">
    <string>Size</string>
   </property>
  </widget>
  <widget class="WidgetTilesetSettings" name="widgetTilesetSettings" native="true">
   <property name="geometry">
    <rect>
     <x>207</x>
     <y>37</y>
     <width>311</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBoxLocation">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>470</y>
     <width>311</width>
     <height>37</height>
    </rect>
   </property>
   <property name="title">
    <string>Location</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>221</x>
     <y>160</y>
     <width>57</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Texture:</string>
   </property>
  </widget>
  <widget class="WidgetPicture" name="widgetPicture" native="true">
   <property name="geometry">
    <rect>
     <x>284</x>
     <y>150</y>
     <width>200</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>27</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>27</height>
    </size>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PanelSuperList</class>
   <extends>QWidget</extends>
   <header>panelsuperlist.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetPicture</class>
   <extends>QWidget</extends>
   <header>widgetpicture.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetTilesetSettings</class>
   <extends>QWidget</extends>
   <header>widgettilesetsettings.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
